modal/dict.py

```diff
@@ -75,13 +76,13 @@ def __init__(self, data={}):
         self._init_from_other(obj)
 
     @staticmethod
-    def persisted(
-        label: str, namespace=api_pb2.DEPLOYMENT_NAMESPACE_WORKSPACE, environment_name: Optional[str] = None
+    def from_name(
+        label: str,
+        namespace=api_pb2.DEPLOYMENT_NAMESPACE_WORKSPACE,
+        environment_name: Optional[str] = None,
+        create_if_missing: bool = False,
```

Comment:
Thoughts on the default here? It feels like you'd usually want this to be `True`, and the main reason you would want `False` would be to avoid bugs from typo-ing the name or something similar. Maybe that's a good reason to set it to `False` in code that you want to be defensive about but I'm not sure if it's the friendliest default.

Reply:
feels like `False` is the most defensive default since it has no side effects